src/vcpkg/binarycaching.cpp

```diff
@@ -201,7 +201,9 @@ namespace
                               .string_arg("-y")
                               .string_arg("-r")
                               .path_arg(destination)
-                              .string_arg("*"),
+                              .string_arg("*")
+                              .string_arg("--exclude")
+                              .string_arg("\\*.DS_Store"),
```

Comment:
Yeah it is done here...

Reply:
... oops, thanks

Reply:
we also need to add this in the other places we create zips - namely, export - and this may mean we should make "create a zip" it's own operation, rather than doing it via the command line whenever we need to.